### PR TITLE
Sprinkle some design over the profile and header

### PR DESCRIPTION
**Pull request by gabebw**
- Header now has link to root path and good-looking, right-floated items
- Profile page includes Gravatar
- Add whitespace in a lot of places

youshouldtotally 2016-01-16 23-08-18

### CODE REVIEW

**Comment by adarsh** [comment]
Screen cap?

On Saturday, January 16, 2016, Gabe Berke-Williams notifications@github.com
wrote:

> - Header now has link to root path and good-looking, right-floated
>   items
> - Profile page includes Gravatar
> - Add whitespace in a lot of places
> 
> ---
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/gabebw/youshouldtotally/pull/8
> Commit Summary
> - Sprinkle some design over the profile and header
> 
> File Changes
> - _M_ Gemfile
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-0 (1)
> - _M_ Gemfile.lock
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-1 (2)
> - _M_ app/assets/stylesheets/application.scss
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-2 (1)
> - _M_ app/assets/stylesheets/base/_container.scss
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-3 (5)
> - _A_ app/assets/stylesheets/components/_header.scss
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-4 (20)
> - _A_ app/assets/stylesheets/components/_recommendations.scss
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-5 (5)
> - _A_ app/assets/stylesheets/pages/_users.scss
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-6 (13)
> - _M_ app/controllers/users_controller.rb
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-7 (4)
> - _M_ app/views/layouts/application.html.erb
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-8 (34)
> - _M_ app/views/users/show.html.erb
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-9 (18)
> - _M_ config/locales/en.yml
>   https://github.com/gabebw/youshouldtotally/pull/8/files#diff-10 (2)
> 
> Patch Links:
> - https://github.com/gabebw/youshouldtotally/pull/8.patch
> - https://github.com/gabebw/youshouldtotally/pull/8.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gabebw/youshouldtotally/pull/8.

## 

## 

@adarshp http://www.twitter.com/adarshp
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal

**Comment by gabebw** [comment]
Added!

File: app/assets/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.header__items {
+  float: right;
+  margin-right: $base-spacing;
+
+  .header__item {
+    display: inline;
+
+    &:not(:last-child):after {
+      content: "|";
```
Comment (adarsh) [inline app/assets/stylesheets/components/_header.scss:17]: Good one

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -17,8 +17,8 @@ def create
   end
 
   def show
-    user = User.find(params[:id])
-    @television_shows = user.television_shows
+    @user = User.find(params[:id])
```
Comment (adarsh) [inline app/controllers/users_controller.rb:20]: Dumb question but this is indexed, right?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -14,21 +14,31 @@
   <%= csrf_meta_tags %>
 </head>
 <body class="<%= body_class %>">
-  <div id="container">
-    <%= render "flashes" -%>
-    <header>
+  <%= render "flashes" -%>
+
+  <header>
```
Comment (adarsh) [inline app/views/layouts/application.html.erb]: Maybe extract a header partial?

**Comment by adarsh** [comment]
Beyooooootiful!

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (gabebw) [inline app/controllers/users_controller.rb:20]: Yep, [primary keys are always indexed](http://www.postgresql.org/docs/current/interactive/sql-createtable.html):

> PostgreSQL automatically creates an index for each unique constraint and primary key constraint to enforce uniqueness. Thus, it is not necessary to create an index explicitly for primary key columns.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (gabebw) [inline app/views/layouts/application.html.erb]: Extracted a header: 55634f31771cc7783701c46c978ddefe15b9f031

**Pull request merged**